test/format/util.c

```diff
@@ -608,25 +609,18 @@ timestamp(void *arg)
 	 */
 	while (!g.workers_finished) {
```

Comment:
Is this loop still necessary?  What stops us using a "get=all_committed" query (under a lock) to update the oldest timestamp during the run?

If we could do that, then there would be no need to track `tinfo->last_timestamp`.

Reply:
I agree we could acquire a lock and use `get=all_committed`, but I think that would have to serialize with transaction-begin and transaction-prepare. I don't have data to show it, but it seems that might result in less thread independence and worse testing.

Just let me know if you prefer the lock.

Reply:
@keithbostic sorry for letting this one sit.  My preference is to try to get all of the timestamp tracking out of test code, so can you please try this with a lock and see whether getting rid of `tinfo->last_timestamp` is feasible?